tests/vmi_multus_test.go

```diff
@@ -215,7 +215,7 @@ var _ = Describe("Multus", func() {
 					&expect.BExp{R: "\\$ "},
 					&expect.BSnd{S: cmdCheck},
 					&expect.BExp{R: "\\$ "},
-					&expect.BSnd{S: "ip addr show eth1 | grep 10.1.1 | wc -l"},
+					&expect.BSnd{S: "ip addr show eth1 | grep 10.1.1 | wc -l\n"},
```

Comment:
Wow... Proof right there that this PR is desperately needed!

Reply:
Added the test id